start.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Run migrations
+poetry run flask db upgrade
+
+# Start the server
+poetry run flask run -p 8080 --host 0.0.0.0
```

Comment:
this runs the app using flask's development server, we probably want to run via gunicorn?

Reply:
Good idea